sys-fs/squashfs_tools/squashfs_tools-4.3.recipe

```diff
@@ -0,0 +1,105 @@
+SUMMARY="Provides tools for creating and manipulating squashfs filesystems"
+DESCRIPTION="squashfs-tools provides tools for creating and appending to \
+(mksquashfs) and extracting (unsquashfs) squashfs filesystems.
+
+Squashfs is a compressed read-only filesystem. It is intended for general \
+read-only filesystem use, archival use (i.e. in cases where a .tar.gz file \
+may be used, and in constrained block device/memory systems (e.g. embedded \
+systems) where low overhead is needed."
+HOMEPAGE="http://squashfs.sourceforge.net/"
+COPYRIGHT="2002-2014 Philip Lougher"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sf.net/squashfs/squashfs$portVersion.tar.gz"
+CHECKSUM_SHA256="0d605512437b1eb800b4736791559295ee5f60177e102e4d4ccd0ee241a5f3f6"
+SOURCE_DIR="squashfs$portVersion/squashfs-tools"
+
+DEBIAN_REV="3"
+SOURCE_FILENAME_2="squashfs-tools_$portVersion-$DEBIAN_REV.debian.tar.xz"
+SOURCE_URI_2="http://httpredir.debian.org/debian/pool/main/s/squashfs-tools/$SOURCE_FILENAME_2"
+CHECKSUM_SHA256_2="1c296cc147e322e7124bf23a3c242ac83f6a986e6d6f64829ad2424b33914fc4"
+
+COMMIT="dceb729f0369d72f1d7820705fd12510b71446d2"
+PATCHES_BASE_URI="https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-fs/squashfs-tools/files"
```

Comment:
Can you please replace
```
COMMIT="dceb729f0369d72f1d7820705fd12510b71446d2"
PATCHES_BASE_URI="https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-fs/squashfs-tools/files"
```
by
```
commit="dceb729f0369d72f1d7820705fd12510b71446d2"
patchesBaseUri="https://gitweb.gentoo.org/repo/gentoo.git/plain/sys-fs/squashfs-tools/files"
```
and change the SOURCE_URI_{3-8} to:
```
SOURCE_URI_3="$patchesBaseUri/$SOURCE_FILENAME_3?id=$commit#noarchive"
SOURCE_URI_4="$patchesBaseUri/$SOURCE_FILENAME_4?id=$commit#noarchive"
SOURCE_URI_5="$patchesBaseUri/$SOURCE_FILENAME_5?id=$commit#noarchive"
SOURCE_URI_6="$patchesBaseUri/$SOURCE_FILENAME_6?id=$commit#noarchive"
SOURCE_URI_7="$patchesBaseUri/$SOURCE_FILENAME_7?id=$commit#noarchive"
SOURCE_URI_8="$patchesBaseUri/$SOURCE_FILENAME_8?id=$commit#noarchive"
```

Thanks!

BTW, here are the reasons for these cosmetic changes:

* We currenly have "COMMIT" for 4 recipes, "commit" for 22 recipes and "srcGitRev" for 42 recipes.

When I started recommending "srcGitRev" sometime ago, I did so because it was already the most used custom variable, but I had not noticed that there were almost 20 "commit"s.
Since "srcGitRev" is a bit strange, I think we should stop using it and start using "commit" instead.

As said earlier, I think we should avoid all-caps in custom variables, so I would recommend to use commit instead of COMMIT.

* haikuporter complains about PATCHES_BASE_URI and says it will ignore it.
So I would recommend to use patchesBaseUri instead of PATCHES_BASE_URI.